lib/deimos/config/phobos_config.rb

```diff
@@ -62,20 +62,46 @@ def phobos_config
         backoff: _backoff(self.consumers.backoff.to_a)
       }
 
-      p_config[:listeners] = self.consumer_objects.map do |consumer|
+      # Phobos will support multiple topics subscriptions per listener
+      topics = self.consumer_objects.map do |consumer|
         next nil if consumer.disabled
+        consumer.topic
+      end.compact
 
-        hash = consumer.to_h.reject do |k, _|
-          %i(class_name schema namespace key_config backoff disabled).include?(k)
-        end
-        hash = hash.map { |k, v| [k, v.is_a?(Symbol) ? v.to_s : v] }.to_h
-        hash[:handler] = consumer.class_name
-        if consumer.backoff
-          hash[:backoff] = _backoff(consumer.backoff.to_a)
-        end
-        hash
-      end
-      p_config[:listeners].compact!
+      # The following settings must move up to app level configs.These are just 
+      # placeholders names / values for now but we'll probably would want to move them
+      # to the Deimos.config object
+      max_bytes_per_partition = 524_288
+      delivery = 'message'
```

Comment:
Loosing granularity on this particular attribute is a bit of a bummer.

Reply:
Actually, looking at RubyKafka, and the Phobos PR, there's no reason we can't specify separate `max_bytes_per_partition` for each topic. Since we're using different `subscribe` calls, and each call takes that as an option, we can at least do it that way.

More important would be the ability to separate out batch from message processing on the Phobos side, because I think it is important to allow each topic to have a separate config for that. I think we need to still enable that in the Phobos config, which means a larger change than that PR has right now.

Reply:
ok, yes this will require that we provide a way to do per topic configuration in phobos. How about supporting something like this:

```yml
listeners:
  - handler: DeimosHandler
     topics: 
       topic1:
         max_bytes_per_partition: 32400
         delivery: batch
       topic2:
         max_bytes_per_partition: 524288
         delivery: message
    topic:
    group_id: test
    max_bytes_per_partition: 32400
    delivery: message
    start_from_beginning: true
```

In the implementation we will then merge the specific topic configuration over the top level (default ones) to override them if they're defined. How does that sounds?